lib/vector-config/src/schema/visitors/unevaluated.rs

```diff
@@ -342,13 +361,35 @@ fn is_markable_schema(definitions: &Map<String, Schema>, schema: &SchemaObject)
                     .as_ref()
                     .and_then(|reference| {
                         let reference = get_cleaned_schema_reference(reference);
-                        definitions.get(reference)
+                        definitions.get(reference).map(|d| (reference, d))
```

Comment:
Would `.get_full` work here instead of needing the `.map`?

Reply:
Ah, nice callout! Yeah, that'd be a little bit cleaner.

Reply:
Fixed in 712ca0d.